ellama.el

```diff
@@ -179,8 +183,16 @@ in.  Default value is (current-buffer).
 :point POINT -- POINT is the point in buffer to insert ellama reaply at."
   (let* ((buffer (or (plist-get args :buffer) (current-buffer)))
 	 (point (or (plist-get args :point)
-		    (with-current-buffer buffer (point)))))
+		    (with-current-buffer buffer (point))))
+	 (filter (or (plist-get args :filter) #'identity))
+	 (session (plist-get args :session))
+	 (errcb (or (plist-get args :on-error) (lambda (msg) (error "Error calling the LLM: %s" msg))))
+	 (donecb (or (plist-get args :on-done) #'ignore)))
     (with-current-buffer buffer
+      (if (and session ellama--chat-prompt)
+	  (llm-chat-prompt-append-response
+	   ellama--chat-prompt prompt)
+	(setq ellama--chat-prompt (llm-make-simple-chat-prompt prompt)))
```

Comment:
This means we always set `ellama--chat-prompt`. Which, IMO, is kind of convenient anyways.